src/main/java/com/efficio/fieldbook/web/common/controller/ExportStudyGermplasmController.java

```diff
@@ -103,7 +107,7 @@ protected String doExport(final int exportType, final HttpServletResponse respon
 
 		final int studyId = this.userSelection.getWorkbook().getStudyDetails().getId();
 		final boolean hasStocks =
```

Comment:
While we're standardizing nomenclature, let's rename:
1. `ExportStudyGermplasmController` -> `ExportStudyEntriesController`
2. `hasStocks` variable -> `studyHasEntries`